tools/wpt/run.py

```diff
@@ -465,6 +465,18 @@ def setup_logging(kwargs, default_config=None):
 
     global logger
 
+    class HasLogHandler(mozlog.handlers.BaseHandler):
+        def __init__(self):
+            self.inner = None
```

Comment:
This feels weird, given we don't really want an inner logger at all, but it's not at all clear what the intended way to write a new handler is ([bugzilla 1515330](https://bugzilla.mozilla.org/show_bug.cgi?id=1515330)).